.github/CODEOWNERS

```diff
@@ -10,7 +10,7 @@
 
 # In each subsection folders are ordered first by depth, then alphabetically.
 # This should make it easy to add new rules without breaking existing ones.
-
+* @dbt-labs/semantic-layer
 
 # Parsing, validations, and protocols
 /dbt_semantic_interfaces/**  @dbt-labs/semantic-layer @QMalcolm @graciegoheen
```

Comment:
Should we remove the team tag from this line then?

Reply:
We can, if you don't anticipate any other teams owning any portion than I will remove it... just lmk